ui/src/components/settings/SettingRow.js

```diff
@@ -120,6 +125,18 @@ export default class SettingRow extends React.Component {
                         />
                     </StyledDiv>
                 );
+            case 'text':
+                return (
+                    <StyledDiv>
+                        <SettingInput
+                            placeholder={this.props.unit}
+                            fluid
```

Comment:
just curious, what is fluid?

Reply:
https://github.com/AthenZ/athenz/blob/master/ui/src/components/denali/InputDropdown.js#L485